tests/e2e/utils/src/flows/with-rest-api.js

```diff
@@ -152,5 +153,31 @@ export const withRestApi = {
 				expect( response.value ).toBe( defaultSetting.value );
 			}
 		}
+	},
+	/**
+	 * Create a batch of orders with the given `statuses`
+	 * using the "Batch Create Order" API endpoint.
+	 * 
+	 * @param statuses Array of order statuses
+	 */
+	batchCreateOrders : async (statuses) => {
+		const defaultOrder = config.get('orders.basicPaidOrder');
+		const path = '/wc/v3/orders/batch';
+
+		// Create an order per status
+		const orders = statuses.map((status) => {
+			return {
+				...defaultOrder,
+				status: status
+			};
+		});
+
+		// Set the request payload from the created orders.
+		// Then send the API request.
+		const payload = { create: orders };
+		const { statusCode } = await client.post(path, payload);
+
+		/* eslint-disable jest/no-standalone-expect */
```

Comment:
@zhongruige @rrennick I had to put this otherwise I get an eslint error when I try to push my changes. I know that we discussed this in Slack and based on my understanding, we will fix this in the coming sprints. But for now, is it ok for us to add `eslint-disable` comments like this, or is there a more elegant way workaround? Thanks!

Reply:
@rodelgc You can remove that line. The jest linting rules were removed in https://github.com/woocommerce/woocommerce/pull/30206.

Reply:
@rodelgc I misread your question. In our other files we have disabled the jest linting at the top of the file. The reason you get that error running the tests is that the function is called inside `beforeAll` vs inside `it`.